config/initializers/webpacker_patch_1399.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+
+Webpacker::Compiler.prepend(Module.new do
+  def compile
+    if stale?
+      record_compilation_digest
+      run_webpack.tap do |success|
+        remove_compilation_digest if !success
+      end
+    else
+      true
+    end
+  end
+
+  private
+
+  def remove_compilation_digest
+    compilation_digest_path.delete if compilation_digest_path.exist?
```

Comment:
Did you see https://github.com/rails/webpacker/commit/d22a3c99b195d43426a148bccb9cebfd6fde7d03? Do we need it here as well?

Reply:
Probably you are right @guicassolato I should have taken the master branch
Good catch thanks!